parseTeXlog.py

```diff
@@ -27,7 +27,7 @@ def advance_iterator(it):
 
 def debug(s):
 	if print_debug:
-		print ("parseTeXlog: " + s.encode('UTF-8')) # I think the ST2 console wants this
+		print ("parseTeXlog: " + s) #s.encode('UTF-8')) # I think the ST2 console wants this
```

Comment:
Is this change necessary for these changes to work? It looks as though it may be harmful e.g. If utf-8 characters somehow make it into one of the output messages.